protoc-gen-swagger/genswagger/template.go

```diff
@@ -292,6 +292,34 @@ func renderMessagesAsDefinition(messages messageMap, d swaggerDefinitionsObject,
 	}
 }
 
+func renderMessagesAsStreamDefinition(messages messageMap, d swaggerDefinitionsObject, reg *descriptor.Registry) {
+	for name, msg := range messages {
+		if skipRenderingRef(name) {
+			continue
+		}
+
+		if opt := msg.GetOptions(); opt != nil && opt.MapEntry != nil && *opt.MapEntry {
+			continue
+		}
+		d[fullyQualifiedNameToSwaggerName(msg.FQMN(), reg)] = swaggerSchemaObject{
+			schemaCore: schemaCore{
+				Type: "object",
+			},
+			Title: fmt.Sprintf("Stream result of %s", fullyQualifiedNameToSwaggerName(msg.FQMN(), reg)),
+			Properties: &swaggerSchemaObjectProperties{
+				keyVal{
+					Key: "result",
+					Value: swaggerSchemaObject{
+						schemaCore: schemaCore{
+							Ref: fmt.Sprintf("#/definitions/%s", fullyQualifiedNameToSwaggerName(msg.FQMN(), reg)),
+						},
+					},
+				},
+			},
```

Comment:
I just realized that this doesn't cover the possibility of an error return; https://github.com/grpc-ecosystem/grpc-gateway/blob/a0500cba2195de914ae92997961c957aac0082f2/runtime/handler.go#L193. We need to add the potential for this to contain an "error" key. This is exclusive with the "result" key, so ideally we'd have some way to express that it will be either "result" or "error". Also note the definition of the error returned, this will probably have to be hardcoded in here: https://github.com/grpc-ecosystem/grpc-gateway/blob/a0500cba2195de914ae92997961c957aac0082f2/runtime/internal/stream_chunk.pb.go#L24:6.